src/WickrIOConfigure.js

```diff
@@ -15,11 +15,18 @@ const {exec, execSync, execFileSync} = require('child_process');
 
 class WickrIOConfigure
 {
-    constructor(tokens, processesFile, supportAdministrators, supportVerification, supportEncrypt)
+    constructor(tokens, processesFile, supportAdministrators, supportVerification, supportEncrypt, addOnToJSON)
     {
         this.supportsVerification = false;
         this.supportsAdministrators = false;
         this.supportsEncrypt = false;
+        this.addOnToJSON = false;
+
+        if (addOnToJSON === undefined || addOnToJSON !== false)
```

Comment:
This doesn't make sense, if the default is false then if the "addOnToJSON" is undefined it should use the default value.